packages/blade/src/components/BladeProvider/BladeProvider.native.tsx

```diff
@@ -19,7 +15,7 @@ const BladeProvider = ({
   const { theme, themeContextValue } = useBladeProvider({ initialColorScheme, themeTokens });
 
   return (
-    <GestureHandlerRootView style={gestureHandlerStyle}>
+    <GestureHandlerRootView style={{ flex: 1 }}>
```

Comment:
This fails on master too.

Reply:
Let's revert this anyways, on the root provider best to avoid plain objects since it might cause unnecessary rerender of whole root if any state changes inside useBladeProvider.